### PR TITLE
Guatemala

### PR DESCRIPTION
**Pull request by aaron-kaplan**

### CODE REVIEW

**Comment by aaron-kaplan** [comment]
@nitinmagima I started with your commit from Bitbucket, and made further changes to get it working. Look at the individual commits to see what I changed from your version.

I also added bad years and ENSO data to the fbf_maproom table in the DesignEngine database.

**Comment by aaron-kaplan** [comment]
Merging now so I can get it on the server in time for tomorrow morning. @nitinmagima you can review after the fact and we can go over the changes together.

File: fbfmaproom/fbfmaproom-sample.yaml [hunk 1]
```diff
@@ -345,3 +345,62 @@ countries:
...
+            - name: National
+              sql: select adm0_code as key, adm0_name as label, ST_AsBinary(the_geom) as the_geom
+                  from iridb.g2015_2012_0 where adm0_name = 'Guatemala'
+              vuln_sql: select adm0_code as key, year, sum(vulnerability) as vuln
+                  from public.fbf_vulnerability where adm0_name = 'Guatemala' group by 1, 2
+
+            - name: Regional
+              sql: select (adm0_code, adm1_code) as key, adm1_name as label,
+                  ST_AsBinary(the_geom) as the_geom from iridb.g2015_2014_1
+                  where adm0_name = 'Malawi'
```
Comment (aaron-kaplan) [inline fbfmaproom/fbfmaproom-sample.yaml:367]: Should have been Guatemala. Fixing on master.

**Comment by nitinmagima** [comment]
Sounds good @aaron-kaplan! Let me know if you have time next week to walk through it

**Pull request merged**